boot/bootutil/src/boot_record.c

```diff
@@ -44,18 +44,8 @@
  */
 static bool shared_memory_init_done;
 
-/**
- * @brief Add a data item to the shared data area between bootloader and
- *        runtime SW
- *
- * @param[in] major_type  TLV major type, identify consumer
- * @param[in] minor_type  TLV minor type, identify TLV type
- * @param[in] size        length of added data
- * @param[in] data        pointer to data
- *
- * @return                0 on success; nonzero on failure.
- */
-static int
+/* See in boot_record.h */
+int
```

Comment:
You should add an abort early check for `data == NULL` to the function as well.

Reply:
Fixed.